templates/register.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+{% from 'bootstrap5/form.html' import render_form, render_field %}
+{% block app_content %}
+    <h1>Регистрация</h1>
+    <div class="container">
+            <form action="" method="post">
+                {{ form.csrf_token() }}
+                {{ render_field(form.username) }}
+                {{ render_field(form.email) }}
+                {{ render_field(form.password) }}
```

Comment:
Что если ваша форма будет занимать десятки строк? Нужно закладывать возможность масштабирования
```suggestion
                {% for field in form if field.name != 'csrf_token' %}
                    {{ render_field(field) }}
                    <ul class="alert alert-danger">
	                {% for error in field.errors %}
	                    <li>{{ error }}</li>
	                {% endfor %}
	            </ul>
	        {% endfor %}
```
Причем строки
```html
                    <ul class="alert alert-danger">
	                {% for error in field.errors %}
	                    <li>{{ error }}</li>
	                {% endfor %}
	            </ul>
```
будут как раз выводить сообщения в случае если форма не пройдет валидацию.